src/openms/include/OpenMS/FORMAT/MRMFeatureQCFile.h

```diff
@@ -54,46 +54,47 @@ namespace OpenMS
     public ProgressLogger
   {
 public:
-  ///Default constructor
-  MRMFeatureQCFile();
-  ///Destructor
-  ~MRMFeatureQCFile() override;
+  /// Default constructor
+  MRMFeatureQCFile() = default;
+  /// Destructor
+  ~MRMFeatureQCFile() = default;
 
   /**
     @brief Loads an MRMFeatureQC file.
 
     @exception Exception::FileNotFound is thrown if the file could not be opened
     @exception Exception::ParseError is thrown if an error occurs during parsing
   */
-  void load(const String & filename, MRMFeatureQC & mrmfqc);
```

Comment:
It will most likely be safer to add in an additional argument that allows the user to specify exactly what file will be loaded in.